README.md

```diff
@@ -16,11 +15,11 @@
 
 ## Challenges:
 
-   - New package manager compatibility with existing dependencies is a challenge
-   - May require updating or replacing existing dependencies to work with a new package manager
-   - Compatibility with other project tools and systems should be considered
-   - Careful review and testing can minimize disruptions during migration
-   - Smooth development workflow can be maintained with proper planning
+   - Swift package manager incompatible dependencies and version conflicts.
```

Comment:
1. It's not conveying what it says? can you please explain this in simple terms
2. From the second point I think instead of specifying the package manager use packages.

Reply:
done